packages/chakra-ui/src/BaseInput/BaseInput.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react'
+import { Input, InputProps } from '@chakra-ui/react'
+import PropTypes from 'prop-types'
+import { WidgetProps } from "@rjsf/core";
+import { JSONSchema7, JSONSchema7Object, JSONSchema7Array } from 'json-schema'
+
+type TWidgetProps = WidgetProps & {
+  schema: JSONSchema7 & {
+    examples: | string
+    | number
+    | boolean
+    | JSONSchema7Object
+    | JSONSchema7Array
+    | string[]
+    | any
+  }
+}
+
+type ExtInputProps = InputProps & {
+  list: string | undefined;
+  // onChange: any;
+  // onBlur: any;
+  // onFocus: any;
+}
+
+type TBaseInput = TWidgetProps & ExtInputProps
+
+
```

Comment:
Notice how when an input has an error, the left side of it seems to be a cut off / too thin like this:

![image](https://user-images.githubusercontent.com/1689183/111926820-bc09b100-8a84-11eb-9c16-0bb415b3dfd9.png)

Could you fix?

(Compare this to how it looks like on the chakra ui website:

![image](https://user-images.githubusercontent.com/1689183/111926841-d2177180-8a84-11eb-8341-87e5057230e0.png)
)